lib/abr/simple_abr_manager.js

```diff
@@ -206,10 +211,18 @@ shaka.abr.SimpleAbrManager = class {
     if (this.resizeObserver_ && this.config_.restrictToElementSize) {
       const devicePixelRatio = this.config_.ignoreDevicePixelRatio ?
           1 : this.windowToCheck_.devicePixelRatio;
-      maxHeight = Math.min(
-          maxHeight, this.mediaElement_.clientHeight * devicePixelRatio);
-      maxWidth = Math.min(
-          maxWidth, this.mediaElement_.clientWidth * devicePixelRatio);
+      if (this.pictureInPictureWindow_ && document.pictureInPictureElement &&
+          document.pictureInPictureElement == this.mediaElement_) {
+        maxHeight = Math.min(
+            maxHeight, this.pictureInPictureWindow_.height * devicePixelRatio);
+        maxWidth = Math.min(
+            maxWidth, this.pictureInPictureWindow_.width * devicePixelRatio);
+      } else {
+        maxHeight = Math.min(
+            maxHeight, this.mediaElement_.clientHeight * devicePixelRatio);
+        maxWidth = Math.min(
+            maxWidth, this.mediaElement_.clientWidth * devicePixelRatio);
+      }
```

Comment:
Please conditionally set width and height only and do calculations outside of if/else clause to minify repeating yourself

Reply:
done!